src/EditorFeatures/Core/FindUsages/AbstractFindUsagesService.ProgressAdapter.cs

```diff
@@ -66,8 +66,7 @@ private class FindReferencesProgressAdapter : IStreamingFindReferencesProgress
             /// This dictionary allows us to make that mapping once and then keep it around for
             /// all future callbacks.
             /// </summary>
-            private readonly Dictionary<ISymbol, DefinitionItem> _definitionToItem =
-                new(MetadataUnifyingEquivalenceComparer.Instance);
+            private readonly Dictionary<SymbolGroup, DefinitionItem> _definitionToItem = new();
```

Comment:
it is intentional that we don't have the equivalence comparer here.  That merging logic moves *into* SymbolGroup, so it doesn't have to be specified here.

Reply:
The new concept in this pr is a 'SymbolGroup'.  Instead of treating all the symbols we find from checking linked-files as independent symbols, we now treat them as one 'symbol group'.  We will show this group as just a single entry in teh FAR UI, though with info to indicate which projects it is valid in.